### ISSUE TITLE
Add an action to notify us update compatible.csv when update the ckb or Neuron major or minor version

### ISSUE THREAD

**Issue by yanguoyu** [Magickbase/neuron-public-issues#303]

**Comment by Keith-CY** [comment]
It can be an action to update the compatibility table, and fixed manually by suggestion of the PR

**Comment by yanguoyu** [comment]
> It can be an action to update the compatibility table, and fixed manually by suggestion of the PR

That's right, we can copy the latest compatibility row by action and fix it manually if it needs to be fixed.
